docs/Displaying a User Interface/Main Menu/index.md

```diff
@@ -8,7 +8,33 @@ You have two different options when creating menus. One is to simply add items t
 Every template has a main menu button. The position of this button varies between templates and cannot be removed from the template. Some OEMs may format certain templates to not display the main menu button if you have no menu items (such as the navigation map view).
 !!!
 
-## Adding Menu Items
+## Setting the Menu Layout
+If connected to a newer head unit (RPC v.6.0+) you may have the option to display menu items as a grid of tiles in addition to the default list layout. To find out if the display supports menu tiles, check the system capability manager's @![iOS]`defaultMainWindowCapability.menuLayoutsAvailable`!@@![android,javaEE,javaSE] `getDefaultMainWindowCapability().getMenuLayoutsAvailable()`!@ property after successfully connecting to the head unit. To set the menu layout using the screen manager, you will need to set the `screenManager`s `menuConfiguration` property.
```

Comment:
```suggestion
On some newer head units, you may have the option to display menu items as a grid of tiles instead of the default list layout. To determine if the head unit supports the tiles layout, check the system capability manager's @![iOS]`defaultMainWindowCapability.menuLayoutsAvailable`!@@![android,javaEE,javaSE] `getDefaultMainWindowCapability().getMenuLayoutsAvailable()`!@ property after successfully connecting to the head unit. To set the menu layout using the screen manager, you will need to set the `screenManager`s `menuConfiguration` property.
```